samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -1384,6 +1391,10 @@ mfxStatus CDecodingPipeline::CreateAllocator()
             MSDK_CHECK_STATUS(sts, "m_hwdev->GetHandle failed");
             sts = m_mfxSession.SetHandle(MFX_HANDLE_VA_DISPLAY, va_dpy);
             MSDK_CHECK_STATUS(sts, "m_mfxSession.SetHandle failed");
+            sts = m_mfxSession.SetFrameAllocator(m_pGeneralAllocator);
+            MSDK_CHECK_STATUS(sts, "m_mfxSession.SetFrameAllocator failed");
+
+            m_bExternalAlloc = true;
```

Comment:
Look into comments on lines 1400-1406 below - they become not relevant considering the change you are making here. Please, remove these comment entirely.

Reply:
Sure. I will remove the comment entirely. Thanks.